src/public/index.js

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import App from 'organisms/cards/SelectCard';
+import App from 'atoms/Layout';
```

Comment:
@drewdrewthis I'm not ripping this out yet because the new dev webpack config that Josh made still uses this as the entry point